src/python_testing/TC_OCC_2_2.py

```diff
@@ -0,0 +1,146 @@
+                asserts.assert_equal(occupancy_sensor_type_dut,
+                                     Clusters.OccupancySensing.Enums.OccupancySensorTypeEnum.kPir, "OccupancySensorType is not PIR")
+            # Bitmap supports US, then OccupancySensorType should be US
+            elif (is_pir_feature_supported != 1) & (is_us_feature_supported == 1) & (is_phy_feature_supported != 1):
+                asserts.assert_equal(
+                    occupancy_sensor_type_dut, Clusters.OccupancySensing.Enums.OccupancySensorTypeEnum.kUltrasonci, "OccupancySensorType is not Ultrasonic")
+            # Bitmap supports PIR and US, then OccupancySensorType should be PIRAndUltrasonic
+            elif (is_pir_feature_supported == 1) & (is_us_feature_supported == 1) & (is_phy_feature_supported != 1):
+                asserts.assert_equal(occupancy_sensor_type_dut, Clusters.OccupancySensing.Enums.OccupancySensorTypeEnum.kPIRAndUltrasonic,
+                                     "OccupancySensorType is not PIRAndUltrasonic")
+            # Bitmap supports Physical Contact, then OccupancySensorType should be Physical Contact
+            elif (is_pir_feature_supported != 1) & (is_us_feature_supported != 1) & (is_phy_feature_supported == 1):
+                asserts.assert_equal(occupancy_sensor_type_dut, Clusters.OccupancySensing.Enums.OccupancySensorTypeEnum.kPhysicalContact,
+                                     "OccupancySensorType is not PhysicalContact")
+            # Bitmap supports Physical Contact and PIR, then OccupancySensorType should be PIR
+            elif (is_pir_feature_supported == 1) & (is_us_feature_supported != 1) & (is_phy_feature_supported == 1):
+                asserts.assert_equal(occupancy_sensor_type_dut,
+                                     Clusters.OccupancySensing.Enums.OccupancySensorTypeEnum.kPir, "OccupancySensorType is not PIR")
+            # Bitmap supports US and Physical Contact, then OccupancySensorType should be US
+            elif (is_pir_feature_supported != 1) & (is_us_feature_supported == 1) & (is_phy_feature_supported == 1):
+                asserts.assert_equal(
+                    occupancy_sensor_type_dut, Clusters.OccupancySensing.Enums.OccupancySensorTypeEnum.kUltrasonic, "OccupancySensorType is not Ultrasonic")
+            # Bitmap supports PIR, US and Physical Contact, then OccupancySensorType should be PIRAndUltrasonic
+            elif (is_pir_feature_supported == 1) & (is_us_feature_supported == 1) & (is_phy_feature_supported == 1):
+                asserts.assert_equal(occupancy_sensor_type_dut, Clusters.OccupancySensing.Enums.OccupancySensorTypeEnum.kPIRAndUltrasonic,
+                                     "OccupancySensorType is not PIRAndUltrasonic")
+
+        else:
+            logging.info("OccupancySensorType mandatory attribute is not supported. Test step skipped")
+            asserts.fail("Missing mandatory attribute OccupancySensorType")

```

Comment:
if attribute is mandatory, this should be an `assert_in` test not a if/else with the else failing.

Reply:
Done.